src/iop/colorbalancergb.c

```diff
@@ -180,7 +180,8 @@ const char *aliases()
 
 const char **description(struct dt_iop_module_t *self)
 {
-  return dt_iop_set_description(self, _("affect color, brightness and contrast"),
+  return dt_iop_set_description(self, _("a color-grading tools improved from ASC CDL"
```

Comment:
A missing space at the end of line. I'll set this as RFC to get comments.

Reply:
"a color-grading tools" is bad English. It should either be "color grading tools" or "a color grading tool". Please can we also avoid words like "improved" and "properly" and concentrate on what it does, rather than how good we think it is.

Reply:
"a" and "tools" is a typo, fixed

Texts were excerpted from user manual, not wrote by me. 
https://docs.darktable.org/usermanual/4.2/en/module-reference/processing-modules/color-balance-rgb/
I'm okay with any other words if some one has better ideas. To separate two modules' description is my propose.

Reply:
how about this:

```
"grading tools using alpha masks rather than ASC CDL\n"
"works in linear rgb and perceptual color space"
```

Reply:
I don't think there's harm in including the word "color" here, and I'm not sure we need to talk about what it _doesn't_ do. How about "color grading tools using alpha masks to split shadows, mid-tones and highlights"?

Reply:
Sure, I'm totally okay with that.

Will that be too many words in a line? I'm not sure if we have a specification about that.